packages/node/package.json

```diff
@@ -21,6 +21,7 @@
     "puppeteer": "^19.4.1"
   },
   "devDependencies": {
-    "webpack-node-externals": "^2.5.2"
+    "webpack-node-externals": "^2.5.2",
+    "string-replace-middleware": "^1.0.2"
```

Comment:
string replace middleware should be a dependencie

Reply:
You're right, I have fixed it